labs/byzantine-api-gateway.md

```diff
@@ -0,0 +1,23 @@
+# Lab Name
+byzantine api gateway
+
+# Short Description
+Enables client access to Hyperledger Fabric blockchain network through RESTful API's.
+
+# Scope of Lab
+Currently client applications can access HLF network Peers using the GoLang, Node, or Java SDK's. This project provides a generalized API gateway that will allow users to query, execute transactions along with querying block storage and configuration using RESTful API's.
+
+This project provides a ready to use solution that developers would otherwise have to implement when trying to provide web clients access to a HLF network.
+
+# Initial Committers
+Enter the Github IDs for the set of initial committers.
+- https://github.com/dpitt
+- https://github.com/jaimeniswonger
+
+
+# Sponsor
+ - https://github.com/vip@dlt.nyc - Chair of Identity WG
+
+# Pre-existing repository
+If you currently have a Github repository that you wish to transfer to the Hyperledger Labs organization, please provide a link here. **NOTE: Please refer to the README for additional information on existing repositories.**
+- https://github.com/in-the-keyhole/byzantine-api-gateway
```

Comment:
DCO signoff is required for all commits. We may not be able to directly transfer this repository unless it is updated to have DCO signoff on all previous commits.